.github/workflows/docs-check.yml

```diff
@@ -33,15 +33,12 @@ jobs:
         with:
           python-version: 3.9
 
-      - name: Run docs
+      - name: Check helm-docs
```

Comment:
There seems to be an issue with this workflow, it isn't showing up in the CI, and the action shows a [failure](https://github.com/rook/rook/actions/runs/6035747927), although I can't see a failure message. I'm guessing there is some whitespace issue that is failing parsing.

Reply:
@travisn  wrote:
> There seems to be an issue with this workflow, it isn't showing up in the CI, and the action shows a [failure](https://github.com/rook/rook/actions/runs/6035747927), although I can't see a failure message. I'm guessing there is some whitespace issue that is failing parsing.

I guess that's the problem. I tried another update to hopefully fix this.

Reply:
not fixed yet, I'm afraid ...

Reply:
really strange: github says there is a syntax error in line 39: https://github.com/rook/rook/actions/runs/6040131032/workflow#L39

Bit it does not say what the error is. And the yaml lint checks passed ...

Reply:
github says there is a syntax error in line 39 of the workflow file but does not say what error. while the yamllint ci workflow succeeded, locally running yamllint yields an error for line 39:

```
39:13     error    syntax error: mapping values are not allowed here (syntax)
```


I don't know how to fix this ...